ci/builders/linux_web_engine.json

```diff
@@ -426,7 +426,7 @@
         },
         {
           "dependency": "firefox",
-          "version": "version:106.0"
+          "version": "version:132.0"
```

Comment:
We'll want to update this page then: https://docs.flutter.dev/reference/supported-platforms